atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/TableMetaDataProvider.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.keyvalue.cassandra;
+
+import com.palantir.atlasdb.keyvalue.api.KeyValueService;
+import com.palantir.atlasdb.keyvalue.api.TableReference;
+import com.palantir.atlasdb.keyvalue.impl.KeyValueServices;
+import com.palantir.atlasdb.table.description.TableMetadata;
+
+public class TableMetaDataProvider {
```

Comment:
why do we need this class? it feels like its purpose is to be used in tests as some way of mocking i.e. not possible to mock static calls, but you don't have any reference to this in your test. Basically I'm questioning whether this indirection is necessary/worth it?/why not just be a private method inside the `CassandraKeyValueServiceImpl`.

Reply:
I've been silly, turns out this class is used inside `RangeLoader` **not** `CassandraKeyValueServiceImpl`. Don't necessarily want to have a reference to KVS inside `RangeLoader` especially when there's a relation the other way around.